frontend/src/pages/agreements/EditAgreement.js

```diff
@@ -48,17 +48,6 @@ const EditAgreement = () => {
     if (errorAgreement) {
         return <div>Oops, an error occurred</div>;
     }
-    if (agreement.status !== "DRAFT" && agreement.status !== "UNDER_REVIEW") {
```

Comment:
do we really want to remove this requirement?

Reply:
There is no agreement-level status any more. I believe Steve said he was working on something to fix it in his current issue. We'll definitely need to figure out how to limit this, but for the time being the purpose here was to remove the agreement level status from everything.

Reply:
yeah, I haven't sorted out how this is supposed to work in the new design yet, but I thought for Cliff's ticket it would just be removed. 

I'd like something at the Agreement level for this, if we are removing status, then I don't know what it is.  It be nice if there was something that encapsulated the BLI.status logic so clients and server and trying to both looking through all the BLIs and compute the same thing.  Can I add something like "is_editible" or "is_locked" in the GET Agreement(s) response?